dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/HttpEndpoint.java

```diff
@@ -38,6 +40,16 @@ public class HttpEndpoint extends ConfigElement {
     private String httpsPort;
     private String protocolVersion;
 
+    /**
+     * @return HTTP options for this configuration
+     */
+    public HttpOptions getHttpOptions() {
+        if (this.httpOptions == null) {
+            this.httpOptions = new HttpOptions();
+        }
+        return this.httpOptions;
+    }
```

Comment:
Should we update the copyright header here?

Reply:
Fixed